animated-base/src/main/java/com/facebook/fresco/animation/factory/AnimatedFactoryV2Impl.java

```diff
@@ -54,17 +54,20 @@
   private @Nullable AnimatedDrawableBackendProvider mAnimatedDrawableBackendProvider;
   private @Nullable AnimatedDrawableUtil mAnimatedDrawableUtil;
   private @Nullable DrawableFactory mAnimatedDrawableFactory;
+  SerialExecutorService mSerialExecutorService;
```

Comment:
can you please add `private @Nullable`

Reply:
Also, I don't think this has to be a `SerialExecutorService` but can be anything. It's just that the default implementation is one. We can just pass down an `ExecutorService` to make this even more flexible.